### DISCUSSION TITLE
Expired certificate

### DISCUSSION THREAD

**Discussion by david-grew-activeviam** [qos-ch/logback#684] [General]
Hi, 

There appears to be an expired certificate on one of the logback servers. To reproduce use: `curl https://159.100.250.151 -v`. This IP was one of the ones provided by DNS, but others do not seem to be failing.

Thanks,
David

**Comment by david-grew-activeviam** [comment]
actually it looks as though it may be a DNS issue. If I do a DNS lookup, only one provider returns that IP address. The others return a different IP that does not have a cert issue.

**Comment by ceki** [comment]
@david-grew-activeviam  Thank you for bringing this problem to our attention. Issue has been resolved.
